pyega3/pyega3.py

```diff
@@ -431,19 +431,19 @@ def download_file(token, file_id, file_size, check_sum, num_connections, key, ou
 
 
 def download_file_retry(
-        creds, file_id, file_name, file_size, check_sum, num_connections, key, output_file, genomic_range_args,
+        creds, file_id, display_file_name, file_name, file_size, check_sum, num_connections, key, output_file, genomic_range_args,
         max_retries, retry_wait):
     time0 = time.time()
     token = get_token(creds)
 
     if file_name.endswith(".gpg"):
-        logging.info("GPG files are not supported")
+        logging.info("GPG files are not supported, please use the Java client - https://ega-archive.org/download/using-ega-download-client")
         return
```

Comment:
@anandmohan777 I don't have access to any datasets on which I can test this functionality. Can you please confirm that this is the behavior:
1. User tries to download a .gpg file.
1. pyega3 returns message to terminal that "GPG files are not supported, please use the Java client - https://ega-archive.org/download/using-ega-download-client"
1. pyega3 exits attempt to download GPG file.
1. pyega3 moves to next file in the download request, if present.

I want to confirm that pyega3 doesn't exit completely if it encounters a GPG file, but does continue to download if there are other files in the request.

Reply:
Yes, I can confirm that it is working the way you mentioned above. It doesn't exit completely if it encounters a GPG file and continues to download remaining files. We can see this happening here in the logs also from the comments provided by @Dona094 on the ticket https://www.ebi.ac.uk/panda/jira/browse/EE-1392